cmd/tailscaled/tailscaled_windows.go

```diff
@@ -264,6 +271,16 @@ func startIPNServer(ctx context.Context, logid string) error {
 	return err
 }
 
+func handleSessionChange(chgRequest svc.ChangeRequest) {
+	if chgRequest.Cmd != svc.SessionChange || chgRequest.EventType != windows.WTS_SESSION_UNLOCK ||
+		!flushDNSOnSessionUnlock {
+		return
+	}
+
+	log.Printf("Received WTS_SESSION_UNLOCK event, initiating DNS flush.")
+	go dns.Flush()
```

Comment:
Can we log if the dns.Flush returns an error (which I think can happen, if ipconfig fails to exec)?

I think this means running an anonymous function as the goroutine:
```
go func() {
        err := dns.Flush()
        ...log on error, etc...
        }()
```

Reply:
Sure, done